lib/servers/datavault/data_vault.py

```diff
@@ -447,6 +447,17 @@ def getTags(self, sessions, datasets):
         dataTags = [(d, sorted(self.dataset_tags.get(d, []))) for d in datasets]
         return sessTags, dataTags
 
+class Image:
```

Comment:
@jayich 

I agree in principle but if I do it for this only and not the other classes it will be unorganized and Im not willing to go down that rabbit hole of changing all of the classes to modules. We can add this to a datavault refactor issue if you like